### PR TITLE
Added functionality to check if a venue is open

### PR DESCRIPTION
**Pull request by TannerBaldus**
Added a is_open method to venue client as well as a unittest file to prove it works. Also updated setup.py to so it's ready to publish an update to PyPi.

### CODE REVIEW

File: test_is_open.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+global KEY
+
+
+class IsOpenTest(object):
+
+	def test_no_data(self):
+		venue_client = VenueApiClient(KEY)
+		venue_id = "50d70de7fbafb112c2de" ##no data
+		self.assertEqual(venue_client.is_open(venue_id,"12:00:00","Monday"))
```
Comment (jansel) [inline test_is_open.py]: Change `assertEqual` to `assertIsNone`

File: test_is_open.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+	def test_open(self):
+		venue_client = VenueApiClient(KEY)
+		venue_id='b7b1644a6bb10dff58bd' ## open Monday 09:00:00 - 15:00:00
+		self,assertTrue(venue_client.is_open(venue_id,"12:00:00","Monday"))
```
Comment (jansel) [inline test_is_open.py]: Comma a typo?

File: test_is_open.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+from locu import VenueApiClient
+import sys
+import unittest
+
+global KEY
+
+
+class IsOpenTest(object):
```
Comment (jansel) [inline test_is_open.py]: Inherit from `unittest.TestCase`?

**Comment by jansel** [comment]
Thank you for the patch TannerBaldus.  I made a few comments.

File: test_is_open.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import unittest
+
+global KEY
+
+
+<<<<<<< HEAD
+class IsOpenTest(unittest.TestCase):
+=======
+class IsOpenTest(object):
+>>>>>>> 926705451e4d72f635a061e8d36169c1dd49cf28
```
Comment (jansel) [inline test_is_open.py]: It looks like you have committed some un-merged changes in this file.

**Comment by TannerBaldus** [comment]
Sorry bout the delay in getting that fixed, kinda got caught up with term projects, finals and whatnot.

**Comment by jansel** [comment]
Thanks again!

**Pull request merged**